README.md

```diff
@@ -31,3 +31,64 @@ Outputs will be built in the `lib` directory in their own workspaces
 ## Demos
 
 [MVP DApp](./packages/samples/mvp-dapp)
+
+## Quick start
+
+#### Enviroment
+
+    node > 18
+
+#### Kuai Installation
+
+First, you need to install `kuai` globally using `npm` or `yarn`, or just use `npx`:
+
+```bash
+# Use npm to install CLI
+$ npm install -g kuai
+
+# OR Use yarn to install CLI
+$ yarn global add kuai
+```
```

Comment:
`npx` is prior to global installation because it won't affect global environment, and always update-to-date.

Reply:
If the developer uses `npx`, it is unnecessary to install globally. Then they can skip this step.